apps/reflect/src/components/auth/AuthLoginButton.tsx

```diff
@@ -23,23 +23,25 @@ const AuthLoginButton: FC<IProps> = ({
       spacing={2}
       direction="row"
       alignItems="center"
-      sx={
-        !disabled
+      sx={{
+        ...(!disabled
           ? {
               cursor: "pointer",
               "&:hover": {
                 backgroundColor: "#202020",
                 borderRadius: "4px",
               },
             }
-          : {
-              cursor: "not-allowed",
-            }
-      }>
+          : { cursor: "not-allowed" }),
```

Comment:
better the icon has grey color if disable state is true i think add grey color on stack fix that

Reply:
also can you please add e ss for last update :)

Reply:
![CleanShot 2023-05-20 at 23 41 37](https://github.com/senchabot-dev/monorepo/assets/13858549/ce5c9c68-bb41-4fc8-a0e0-8de1a6d09efe)

Reply:
yes 👍🏼